src/main/java/org/fundacion/jala/converter/view/compiler/CodeTextArea.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package org.fundacion.jala.converter.view.compiler;
+
+import javax.swing.JPanel;
+import javax.swing.JTextArea;
+import javax.swing.JScrollPane;
+import javax.swing.border.EmptyBorder;
+import javax.swing.event.DocumentEvent;
+import javax.swing.event.DocumentListener;
+import javax.swing.text.Element;
+
+import java.awt.Color;
+import java.awt.BorderLayout;
+
+class CodeTextArea extends JPanel {
+    private JTextArea codeArea;
+    private JTextArea lineCode;
+
+    /**
+     * Initialize the graphic elements for Code Text Area.
+     */
+    protected CodeTextArea() {
+        codeArea = new JTextArea();
+        codeArea.setBorder(new EmptyBorder(5, 10, 0, 10));
+
+        lineCode =  new JTextArea("1");
+        lineCode.setBorder(new EmptyBorder(5, 10, 0, 10));
+        lineCode.setBackground(Color.darkGray);
+        lineCode.setForeground(Color.white);
+
+        JScrollPane textCodeArea = new JScrollPane();
+        textCodeArea.setBorder(new EmptyBorder(0, 0, 0, 0));
+
+        codeArea.getDocument().addDocumentListener(new DocumentListener() {
+            public String getText() {
+                int caretPosition = codeArea.getDocument().getLength();
```

Comment:
these spaces in the code should be removed

Reply:
Solved 👍